layouts/shortcodes/kf-latest-version.html

```diff
@@ -1 +1 @@
-v0.7.1
\ No newline at end of file
+v1.0-rc.3
```

Comment:
Remove the empty line at the end of this file, and all the other shortcodes. (We're using shortcodes instead of parameters, which makes for easier reading of the Markdown files, but it means we have to avoid empty lines because they cause weird formatting.)

Reply:
Done.